docker-for-mac/apple-m1.md

```diff
@@ -12,13 +12,13 @@ Welcome to Docker Desktop for Apple Silicon.
 >
 > We encourage you to try the release candidate and report any issues in the [Docker Desktop for Mac GitHub](https://github.com/docker/for-mac) repository.
 
-## Docker Desktop RC 1
+## Docker Desktop RC 2
 
-2021-03-18
+2021-03-25
 
 Click on the following link to download the latest release candidate of Docker Desktop for Apple Silicon.
 
-> [Download](https://desktop.docker.com/mac/stable/arm64/62029/Docker.dmg)
+> [Download](https://desktop.docker.com/mac/stable/arm64/62293/Docker.dmg)
 
 ### Known issues
```

Comment:
Delete "we are working with our partners on these".

Then above the VPN bullet, add new rubric:

The following issues are seen when using the hypervisor.framework back end.

Reply:
I think "hypervisor.framework" should be "virtualization.framework" here (qemu uses the "hypervisor.framework" and avoids the issues)